packages/server/graphql/private/mutations/sendBatchNotificationEmails.ts

```diff
@@ -53,13 +61,17 @@ const sendBatchNotificationEmails: MutationResolvers['sendBatchNotificationEmail
   // :TODO: (jmtaber129): Filter out users whose only notification is a team invitation
 
   await Promise.all(
-    users.rows.map((user) => {
-      const {email, preferredName} = user
+    users.rows.map(async (user) => {
+      const {email, tms, preferredName} = user
       const notificationCount = userNotificationMap.get(user.id)!
-      const {subject, html, body} = notificationSummaryCreator({
+
+      const authToken = new AuthToken({sub: user.id, tms, rol: 'impersonate'})
+      const environment = new ServerEnvironment(authToken, dataLoader.share())
```

Comment:
what's the reason for providing an `impersonate` role token here? what change to SSR queries requires that?

Reply:
The thinking here is that since it's not the actual user performing the action, it's not accurate to use a non-impersonating auth token. Not actually needed to make the SSR queries work, but more representative of what the auth token actually is.

Reply:
got it, thanks!